src/main/python/cfn_sphere/aws/kms.py

```diff
@@ -22,7 +21,7 @@ def decrypt(self, encrypted_value):
 
     def encrypt(self, key_id, cleartext_string):
         try:
-            response = self.client.encrypt(KeyId=key_id, Plaintext=binary_type(cleartext_string, 'utf-8'))
```

Comment:
Thanks for that. Did you really test this for python2 and python3? I introduced this to circumvent issues caused by the byte vs str stuff.